modules/monitoring/application/views/scripts/show/components/downtime.phtml

```diff
@@ -61,20 +61,12 @@ if (empty($object->comments) && ! $addLink) {
                         $this->timeUntil($downtime->is_flexible ? $downtime->scheduled_end : $downtime->end)
```

Comment:
This still also needs the new parameter.

Reply:
Done.